src/agent/src/image.rs

```diff
@@ -63,6 +63,27 @@ impl ImageService {
             image_client.config.auth = true;
         }
 
+        #[cfg(feature = "guest-pull")]
+        {
+            let enable_signature_verification = &AGENT_CONFIG.enable_signature_verification;
+            debug!(sl(), "Enable image signature verification: {:?}", enable_signature_verification);
+            image_client.config.security_validate = enable_signature_verification.clone();
+        }
+
+        #[cfg(feature = "guest-pull")]
+        if !&AGENT_CONFIG.image_policy_file.is_empty() {
```

Comment:
No need to take a reference as mentioned in https://github.com/kata-containers/kata-containers/pull/9904#discussion_r1679025221.

Reply:
Yea, thanks!